### ISSUE TITLE
Sketch window dimensions off in Java2D

### ISSUE THREAD

**Issue by shiffman** [processing/processing#3129]
Only tested on Mac OSX retina, 10.10.2

JAVA2D:
![screen shot 2015-02-26 at 11 52 08 am](https://cloud.githubusercontent.com/assets/191758/6391440/2db80752-bdae-11e4-91a4-07f2f1a765ca.png)

P2D:
![screen shot 2015-02-26 at 11 52 22 am](https://cloud.githubusercontent.com/assets/191758/6391443/33a89c6c-bdae-11e4-9195-dc1382fed19f.png)

Code:

```
void setup() {
  size(200, 200);
  //size(200, 200, P2D);
}

void draw() {
  background(0);
  fill(255);
  ellipse(100, 100, 100, 100);
}
```

**Comment by Akarshit** [comment]
Not occuring in linux. Something specific to Mac .

**Comment by shiffman** [comment]
I neglected to mention that I'm testing in the current build 0233.

**Comment by benfry** [comment]
I've been seeing this too. More fixing to do; thanks for filing.

**Comment by benfry** [comment]
Fixed for 3.0a6 with help from #3129.
